packages/vscode-extension/src/chat/packages.ts

```diff
@@ -0,0 +1,83 @@
+import * as vscode from "vscode";
+import { RelativePattern } from "vscode";
+import { Logger } from "../Logger";
+import { findAppRootFolder } from "../utilities/extensionContext";
+import { calculateMD5 } from "../utilities/common";
+import { CHAT_LOG } from ".";
+
+interface Package {
+  name: string;
+  version: string;
+}
+
+const appRootFolder = findAppRootFolder() ?? "";
```

Comment:
we recently refactored some bits to no longer assume appRootFolder is a global constant. It'd be better to get it from `Project` instead and only when it is needed rather than storing the copy